firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/type/Candidate.kt

```diff
@@ -27,7 +34,18 @@ internal constructor(
   public val finishReason: FinishReason?
 )
 
-/** Safety rating corresponding to a generated content. */
+/**
+ * An assessment of the potential harm of some generated content.
+ *
+ * The rating will be restricted to a particular [category].
+ *
+ * @property category The category of harm being assessed (e.g., Hate speech).
+ * @property probability The likelihood of the content causing harm.
+ * @property probabilityScore A numerical score representing the probability of harm.
```

Comment:
Are these scores 0-1? That's my assumption but it'd be nice to have it documented.